init.lua

```diff
@@ -417,6 +417,7 @@ require('lazy').setup({
       'WhoIsSethDaniel/mason-tool-installer.nvim',
 
       -- Useful status updates for LSP.
+      -- NOTE: wow i can type a note too look at mee ooooo
```

Comment:
Does this note serve a purpose?

Reply:
I apologize for the trouble! I must have accidentally copied over my modified init.lua (which included the nerd font change and me testing out the note feature) instead of the one with only the code rearrangement. I'll make another pull request with the latter file.